com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -94,6 +94,8 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem, IProfilableTr
         /// </summary>
         public NetworkSpawnManager SpawnManager { get; private set; }
 
+        internal readonly InternalMessageHandler MessageHandler = new InternalMessageHandler();
```

Comment:
Should this be instantiated within the Init method of the NetworkMananger?

Reply:
Ok...that makes sense.  The task is to remove Singleton references and that is a fair point.